.circleci/config.yml

```diff
@@ -245,9 +252,11 @@ jobs:
   # Note: the image used for this job is built from a docker file found in ../dockerfiles/test
   test_end_to_end: # slower but more in-depth than the test job. Wait until these have passed before deploying the client
     docker:
-      - image: eaadtbs/ib-ci-test:1.4
+      - image: eaadtbs/ib-ci-cypress-test:latest
```

Comment:
Was there actually something we needed to change in the image, in the end?

Reply:
Yeah the image uses cypress:base as the base image and everything else is built on top of it. Without a cypress base image there were some missing dependencies.

Reply:
Ok. We were using `ib-ci-test` for multiple jobs. If the cypress version works for all of those and isn't much of a spin-up time difference, we can reuse it for other jobs too. Otherwise, we'll need to keep the old test image dockerfile around (currently the cypress image replaces it).